governance/README.md

```diff
@@ -1,10 +1,10 @@
 # Governance
 
-The governance of the SAFE Working Group (WG) is facilited by
-Chairs, Technical Leads, and an active group members (see [roles](roles.md)).
-
-Most of the work of the group is conducted by members outside of WG meetings
-(see [process](process.md)). WG meetings serve as a forum for creating a shared
-understanding of the cloud native security landscape with regular updates by
-WG members on project progress and their own security-related, with on-going
-discovery through guest presentations.
\ No newline at end of file
+SIG-Security is a [CNCF Special Interest Group](https://github.com/cncf/toc/tree/master/sigs).
+
+* [Charter](charter.md) - mission and scope
+* [Roles](roles.md) - the work of the group is facilited by Chairs, Technical Leads, and active group members
+* [Process](process.md) - how projects are proposed and work is tracked
+
+General meetings are posted on the CNCF calendar and serve as a forum for creating a shared understanding of the cloud native security landscape with regular updates by members on project progress and their own security-related, with on-going discovery through guest presentations.
```

Comment:
TODO: Are we on the official CNCF Calendar now or are we still on my personal calendar?

Reply:
asked Taylor to put on official calendar -- sorry thought that had already happened!

Reply:
we are now on the official calendar!